chapters/connectors.tex

```diff
@@ -671,7 +671,7 @@ \subsection{Balancing Restriction and Size of Connectors}\label{balancing-restri
 \end{lstlisting}
 \end{example}
 
-\section{Overconstrained Connection-Based Equation Systems}\label{equation-operators-for-overconstrained-connection-based-equation-systems1}\label{overconstrained-connection-based-equation-systems}
+\section{Overconstrained Connectors}\label{equation-operators-for-overconstrained-connection-based-equation-systems1}\label{overconstrained-connectors}
```

Comment:
```suggestion
\section{Overconstrained Connections}\label{equation-operators-for-overconstrained-connection-based-equation-systems1}\label{overconstrained-connections}
```
To me it doesn't make sense to say that a connector is overconstrained in itself. (At most it is "constrained").
Especially as the problem only occurs if there are loops.

Clearly the previous variant was a bit too much, and I think that "Overconstrained Connections" is both short and somewhat accurate.

Reply:
(Ah well, and then we define 'overconstrained connectors' later on. I still believe we shouldn't use it here, and that we maybe later rename them.)

Reply:
I agree that the connectors by themselves are only constrained, so I'll just go with your suggestion for now.